Procfile

```diff
@@ -1 +1 @@
-web: ROCKET_PORT=$PORT ROCKET_KEEP_ALIVE=0 ./target/release/starter-snake-rust
```

Comment:
These options are moved to application logic in `main.rs` or in the configuration file `Rocket.toml`